a/amazon-metadata.yml

```diff
@@ -0,0 +1,19 @@
+amazon-metadata:
+  image: rancher/os-amazon-metadata:v0.9.2${SUFFIX}
```

Comment:
Currently the only reason to rebuild the image is if AWS tools or metadata layout will be changed breaking backward compatibility. But calls I use are very basic for AWS utils and should not change for ages. So I think it should be a separate version.